From e748f0a645438a2ecbdb171964f2b88b93ded738 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Mon, 14 Apr 2014 16:39:06 +0100
Subject: Install bash completion in a file matching the executable name

This allows dynamic completion loading to work properly.

Author: Colin Watson <cjwatson@debian.org>
Forwarded: no
Last-Update: 2014-04-14

Patch-Name: bash-completion-name.patch
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 24fb8c5..5109b2b 100644
--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@ install-pass:
 	@mkdir -p "$(DESTDIR)$(BINDIR)" "$(DESTDIR)$(LIBDIR)" "$(DESTDIR)$(MANDIR)/man1" "$(DESTDIR)$(PREFIX)/share/bash-completion/completions/" "$(DESTDIR)$(PREFIX)/share/zsh/vendor-completions"
 	@install -m 0755 -v src/password-store.sh "$(DESTDIR)$(BINDIR)/pass"
 	@install -m 0644 -v man/pass.1 "$(DESTDIR)$(MANDIR)/man1/pass.1"
-	@install -m 0644 -v src/completion/pass.bash-completion "$(DESTDIR)$(PREFIX)/share/bash-completion/completions/password-store"
+	@install -m 0644 -v src/completion/pass.bash-completion "$(DESTDIR)$(PREFIX)/share/bash-completion/completions/pass"
 	@install -m 0644 -v src/completion/pass.zsh-completion "$(DESTDIR)$(PREFIX)/share/zsh/vendor-completions/_pass"
 
 #	Uncomment to install the fish completion file.
